Plugins/Animation/src/AnimationSystem.cpp

```diff
@@ -118,7 +152,7 @@ void AnimationSystem::handleAssetLoading( Ra::Engine::Entity* entity,
         }
 
         // FIXME(Charly): Certainly not the best way to do this
```

Comment:
What do we do with this FIXME(Charly) ?

Reply:
I don't know either, will take a look at when it was posted